tests/generic_config_updater/test_eth_interface.py

```diff
@@ -256,6 +257,10 @@ def test_replace_fec(duthosts, rand_one_dut_hostname, ensure_dut_readiness, fec)
             current_status_fec = check_interface_status(duthost, "FEC")
             pytest_assert(current_status_fec == fec,
                           "Failed to properly configure interface FEC to requested value {}".format(fec))
+
+            # The rollback after the test cannot revert the fec, when fec is not configured in config_db.json
```

Comment:
@byu343 before do config reload, please check the interface status and fail the case if any interface down, ensure we can capture this unnoticed issue.

Reply:
> @byu343 before do config reload, please check the interface status and fail the case if any interface down, ensure we can capture this unnoticed issue.

@ZhaohuiS Interface down is as expected here, because the test only change FEC on one side, and the other side is using other type of FEC. From my understanding, the test only wants to check that the config is changed but does not care about if the interface is still up after changing the config.